app/ui/Bar/Sidebar.tsx

```
@@ -1,48 +1,72 @@
 
 export function Sidebar() {
   const activeSegment = useSelectedLayoutSegment();
+  const router = useRouter();
+  const { logout } = useLogout();
 
   const activeLinkClassName = "font-bold text-primary";
+
+  const handleLogout = async () => {
+    try {
+      await logout();
+      console.log("로그아웃 성공");
```

ESLint

[error] Unexpected console statement.

(no-console)